sig-cluster-lifecycle/annual-report-2022.md

```diff
@@ -73,6 +102,7 @@ Include any other ways you measure group membership
 
   - cluster-api-operator
   - cluster-api-provider-cloudstack
+  - cluster-api-addon-provider-helm
   - kOps
```

Comment:
why is KOps listed as new?

Reply:
This list is autogenerated when the skeleton of this doc is created, and given that kops has been renamed into kOps,  it is both in new (as a KOps) and deleted (as a kops).